graphql/package.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
I'm a bit inclined to call this directory `graphql-server`.  To make it more clear this is a separate running component.

Reply:
I wonder if we will need changes to our `Dockerfile` as well, to eliminate this:

```
CMD [ "/opt/bridge/bin/bridge", "--public-dir=/opt/bridge/static" ]
```
To allow for mulitple entrypoints in our image.

Reply:
Though, it looks like we already have it in our deployment

https://github.com/openshift/console-operator/blob/master/pkg/console/subresource/deployment/deployment.go#L275